src/main.ts

```diff
@@ -28,7 +65,8 @@ class Crumbs extends EventEmitter {
       this.banner = document.querySelector(".crumbs-banner");
 
       // Clicking on accept all sets all the cookies and hides the banner
-      const acceptCookies = document.querySelector(".crumbs-accept-all");
+      const acceptCookies: HTMLButtonElement =
+        document.querySelector(".crumbs-accept-all");
```

Comment:
I would recommend turning on `"strict": true` in the `tsconfig.json` to get the most bug-catching abilities from TypeScript. If you do this you will get a few new errors when you build.

In this case, for example, you would get an error with strict enabled because `querySelector` could return `null`, which is not compatible with `HTMLButtonElement`.

It is pretty much guaranteed the button exists in this case though, so if you want to avoid the `if (acceptCookies !== null) { }` checks you can tell TypeScript you know better like this:

```
const acceptCookies = document.querySelector(".crumbs-accept-all") as HTMLButtonElement;
```

The "correct" way would probably be to use a generic

```
const acceptCookies = document.querySelector<HTMLButtonElement>(".crumbs-accept-all");

if (acceptCookies !==  null) {
  // do your thing
}
```